sway-ast/src/keywords.rs

```diff
@@ -142,93 +146,121 @@ macro_rules! define_token (
+define_token!(AddEqToken, "`+=`", "+=", [Add, Equals], []);
+define_token!(SubEqToken, "`-=`", "-=", [Sub, Equals], []);
+define_token!(StarEqToken, "`*=`", "*=", [Star, Equals], []);
+define_token!(DivEqToken, "`/=`", "/=", [ForwardSlash, Equals], []);
+define_token!(ShlEqToken, "`<<=`", "<<=", [LessThan, LessThan, Equals], []);
+define_token!(
+    ShrEqToken,
+    "`>>=`",
+    ">>=",
+    [GreaterThan, GreaterThan, Equals],
+    []
+);
 define_token!(
     FatRightArrowToken,
     "`=>`",
+    "=>",
     [Equals, GreaterThan],
     [GreaterThan, Equals]
 );
-define_token!(DotToken, "`.`", [Dot], []);
-define_token!(DoubleDotToken, "`..`", [Dot, Dot], [Dot]);
-define_token!(BangToken, "`!`", [Bang], [Equals]);
-define_token!(PercentToken, "`%`", [Percent], []);
-define_token!(AddToken, "`+`", [Add], [Equals]);
-define_token!(SubToken, "`-`", [Sub], [Equals]);
+define_token!(DotToken, "`.`", ".", [Dot], []);
+define_token!(DoubleDotToken, "`..`", "..", [Dot, Dot], [Dot]);
+define_token!(BangToken, "`!`", "!", [Bang], [Equals]);
+define_token!(PercentToken, "`%`", "%", [Percent], []);
+define_token!(AddToken, "`+`", "+", [Add], [Equals]);
+define_token!(SubToken, "`-`", "-", [Sub], [Equals]);
 define_token!(
     ShrToken,
     "`>>`",
+    ">>",
     [GreaterThan, GreaterThan],
     [GreaterThan, Equals]
 );
-define_token!(ShlToken, "`<<`", [LessThan, LessThan], [LessThan, Equals]);
-define_token!(AmpersandToken, "`&`", [Ampersand], [Ampersand]);
-define_token!(CaretToken, "`^`", [Caret], []);
-define_token!(PipeToken, "`|`", [Pipe], [Pipe]);
+define_token!(
+    ShlToken,
+    "`<<`",
+    "<<",
+    [LessThan, LessThan],
+    [LessThan, Equals]
+);
+define_token!(AmpersandToken, "`&`", "&", [Ampersand], [Ampersand]);
+define_token!(CaretToken, "`^`", "^", [Caret], []);
+define_token!(PipeToken, "`|`", "|", [Pipe], [Pipe]);
 define_token!(
     DoubleEqToken,
     "`==`",
+    "==",
     [Equals, Equals],
     [Equals, GreaterThan]
 );
-define_token!(BangEqToken, "`!=`", [Bang, Equals], [Equals, GreaterThan]);
+define_token!(
+    BangEqToken,
+    "`!=`",
+    "!=",
+    [Bang, Equals],
+    [Equals, GreaterThan]
+);
 define_token!(
     GreaterThanEqToken,
     "`>=`",
+    ">=",
     [GreaterThan, Equals],
     [Equals, GreaterThan]
 );
 define_token!(
     LessThanEqToken,
     "`<=`",
+    "<=",
     [LessThan, Equals],
     [Equals, GreaterThan]
 );
 define_token!(
     DoubleAmpersandToken,
     "`&&`",
+    "&&",
     [Ampersand, Ampersand],
     [Ampersand]
 );
-define_token!(DoublePipeToken, "`||`", [Pipe, Pipe], [Pipe]);
-define_token!(UnderscoreToken, "`_`", [Underscore], [Underscore]);
-define_token!(HashToken, "`#`", [Sharp], []);
-define_token!(HashBangToken, "`#!`", [Sharp, Bang], []);
+define_token!(DoublePipeToken, "`||`", "||", [Pipe, Pipe], [Pipe]);
+define_token!(UnderscoreToken, "`_`", "_", [Underscore], [Underscore]);
+define_token!(HashToken, "`#`", "#", [Sharp], []);
```

Comment:
Should this be changes to the following so that `HashBangToken` can match?

    define_token!(HashToken, "`#`", "#", [Sharp], [Bang]);